product_landing_page/index.css

```diff
@@ -127,6 +128,27 @@ body {
   background-color: darkorange;
 }
 
+.advantages-section {
+  margin: 4rem 0;
+  display: grid;
+  max-width: 50rem;
+  grid-template-columns: 4rem 1fr;
+  grid-column-gap: 3rem;
```

Comment:
Jeżeli atrybuty `grid-column-gap` i dla `grid-row-gap` mają te same wartości, to można to skrócić do `grid-gap` :)